x/mint/minter_test.go

```diff
@@ -95,8 +96,36 @@ func BenchmarkBlockProvision(b *testing.B) {
 	r1 := rand.New(s1)
 	minter.AnnualProvisions = sdk.NewDec(r1.Int63n(1000000))
 
-	// run the Fib function b.N times
+	// run the BlockProvision function b.N times
 	for n := 0; n < b.N; n++ {
 		minter.BlockProvision(params)
 	}
 }
+
+// Next inflation benchmarking
+// BenchmarkNextInflation-4 1000000 1828 ns/op
+func BenchmarkNextInflation(b *testing.B) {
+	minter := InitialMinter(sdk.NewDecWithPrec(1, 1))
+	params := DefaultParams()
+	bondedRatio := sdk.NewDecWithPrec(1, 1)
+
+	// run the NextInflationRate function b.N times
+	for n := 0; n < b.N; n++ {
+		minter.NextInflationRate(params, bondedRatio)
+	}
+
+}
+
+// Next annual provisions benchmarking
+// BenchmarkNextAnnualProvisions-4 5000000251 ns/op
```

Comment:
```suggestion
// BenchmarkNextAnnualProvisions-4 278 ns/op
```

Reply:
lol this is also crazy to add on 5 seconds per block if this is true...
I just ran the benchmarks locally and this is what I got though
```
BenchmarkBlockProvision-4         	 3000000	       519 ns/op
BenchmarkNextInflation-4          	 1000000	      1777 ns/op
BenchmarkNextAnnualProvisions-4   	 5000000	       278 ns/op
```

Reply:
Haha, I forgot a space, the benchmark was `251 ns/op`. Fixed.